heartbeat/aws-vpc-move-ip

```diff
@@ -261,10 +265,19 @@ ec2ip_monitor() {
 ec2ip_drop() {
 	cmd="ip addr delete ${OCF_RESKEY_ip}/32 dev $OCF_RESKEY_interface"
 	ocf_log debug "executing command: $cmd"
-	$cmd
+	if [ "$__OCF_ACTION" = "start" ]; then
+		ocf_log debug "$($cmd 2>&1)"
+	else
+		$cmd
+	fi
 	rc=$?
```

Comment:
Need to fix this. Looks like in a start action, `rc` would be set to the return code of `ocf_log`, not the return code from `$cmd 2>&1`.

Reply:
Do we even **need** to call `ec2ip_drop()` from within the start function? I guess it helps ensure we get the IP addresses and routing table into a fresh state. I don't think that's necessary under normal circumstances, and it might not be necessary under any circumstances.

Considering going a step further than @gguifelixamz 's suggestion, and getting rid of all output from this function during a start.

Reply:
I ended up keeping the `ec2ip_drop()` call and the output (at debug level) just in case of unanticipated edge cases.

`ec2ip_get_and_configure()` **should** never be called if the IP address is configured. The one possible exception I can think of: if the IP address is configured but the link is down. Then we do end up in `get_and_configure() -> drop()`. (Ultimately that start operation will fail though, because the second monitor call within the start function will fail since the link is down.)